### PR TITLE
[PLATFORM-1221] Fix for Stream partition implementation

### PR DESCRIPTION
**Pull request by juhah**
Change partitions field to be part of the stream info section. 

Screen Shot 2020-02-25 at 12 12 56

This was not in the design but for mobile, the field gets pushed to its own line:
Screen Shot 2020-02-25 at 12 13 02

Info icon clicked:
Screen Shot 2020-02-25 at 12 12 32

Tablet view tooltip is right aligned to avoid clipping (also not in design):
Screen Shot 2020-02-25 at 12 12 43

Mobile:
Screen Shot 2020-02-25 at 12 12 48

Tooltip closes after you click outside of it.

### CODE REVIEW

File: app/src/shared/components/SvgIcon/index.jsx [hunk 1]
```diff
@@ -509,10 +509,12 @@ type Props = {
...
 }
 
-const SvgIcon = ({ name, ...props }: Props) => React.cloneElement(sources[name], {
+const SvgIcon = (React.forwardRef(({ name, ...props }: Props, ref) => React.cloneElement(sources[name], {
     ...props,
-})
+    ref,
+})): ComponentType<Props>)
 
+// $FlowFixMe suppress flow complaining about property `names` is missing in  `React.AbstractComponentStatics`
```
Comment (juhah) [inline app/src/shared/components/SvgIcon/index.jsx]: @mondoreale Any idea of what Flow magic to use to fix this 🎆

File: app/src/userpages/tests/components/KeyField/KeyField.test.jsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ describe('KeyField', () => {
                 value="testValue"
             />)
 
-            assert(el.find('Label').text() === 'myKey')
+            assert(el.find('Label').text().includes('myKey'))
```
Comment (juhah) [inline app/src/userpages/tests/components/KeyField/KeyField.test.jsx]: There was an invisible character inside the `Label` that broke tests.

File: app/src/userpages/components/StreamPage/Show/InfoView/index.jsx [hunk 3]
```diff
@@ -74,9 +79,57 @@ export const InfoView = ({ disabled }: Props) => {
...
+    useEffect(() => {
+        if (!tooltipRef.current || !iconRef.current) { return }
+
+        if (tooltipOpen) {
+            const clickHandler = (e: SyntheticInputEvent<EventTarget>) => {
+                e.preventDefault()
+                e.stopPropagation()
+
+                // $FlowFixMe Flow doesn't understand that tooltipRef and iconRef are defined
+                if (tooltipOpen && !tooltipRef.current.contains(e.target) && !iconRef.current.contains(e.target)) {
```
Comment (juhah) [inline app/src/userpages/components/StreamPage/Show/InfoView/index.jsx]: Couldn't get this to work without `$FlowFixMe` 🤷‍♂

**Comment by mattatgit** [comment]
@juhah Ok, must have missed the responsive designs for this second pass on the Partitions stuff, must have been done in a hurry. I'll add those in & post back here.

**Comment by mattatgit** [comment]
Actually, maybe it's not worth spending time to correct the responsive designs for this, because we will have to change this section once this new idea of stream resource locators is introduced. So maybe we'll just have to go with it is as for now. Looks fine for desktops & that's what matters here.

**Pull request merged**